api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -67,12 +68,12 @@ public boolean equals(Object o) {
 
     DbInstitutionEmailAddress that = (DbInstitutionEmailAddress) o;
 
-    return Objects.equals(institution, that.institution)
-        && Objects.equals(emailAddress, that.emailAddress);
+    return Objects.equals(emailAddress, that.emailAddress)
```

Comment:
reordered so the more-likely-to-differ field comes first

Reply:
I usually try to use `expected == actual`. But I'm not that consistent (unless `expected` is a constant`).

Reply:
I'm confused here.  This is not a test so we don't have the concept of expected.  This also needs to use equals() instead of ==

Reply:
yeah never mind. it's an old convention from a different language anyway.